### ISSUE TITLE
[Toggle]: WCAG angular issues

### ISSUE THREAD

**Issue by tobre6** [TEDI-Design-System/angular#42]
## Problem
9.4.1.2 - Puudub komponendi nimi.

**Comment by mart-sessman** [comment]
Needs more info, might be usage problem.
For accessibility, "name" argument is not required
Toggle already supports "id" prop and the component should be labelled using HTML label element. "sr-only" class can also be used if label must be visually hidden
